src/pages/monitoring/overview/index.tsx

```diff
@@ -1,30 +1,59 @@
-import React, { useEffect } from "react";
-import { MDBCard, MDBCardBody } from "mdb-react-ui-kit";
+import { MDBCard, MDBCardBody, MDBCol, MDBRow, MDBBtn } from "mdb-react-ui-kit";
 import "./style.css";
-import { useStore } from "../../../store";
 
-type Theme = "light" | "dark";
+interface OverviewHeaderProps extends React.HTMLAttributes<HTMLDivElement> {
+  children?: React.ReactNode;
+  theme: string;
```

Comment:
instead of string you can pass dark|light

Reply:
Theme is a parameter which Overview receives